dataset/test/dataset_arithmetic_test.cpp

```diff
@@ -348,49 +348,6 @@ TYPED_TEST(DatasetBinaryEqualsOpTest, rhs_DatasetView_coord_mismatch) {
                except::CoordMismatchError);
 }
 
-TYPED_TEST(DatasetBinaryEqualsOpTest,
```

Comment:
No we do not. I *think* I have equivalents for all the *relevant* ones, but there one you comment on here is actually an example of one that is removed: I believe it is not necessary with the new style of event data. Since it is handled entirely based on the `Variable` level there should be no relevant thing to test for binary ops of `Dataset` and `DataArray` (provided that the interface of `Variable` is not broken). This is an example of how the new event data support reduces complexity of `Dataset`.